mucommander-translator/src/main/resources/dictionary_pl.properties

```diff
@@ -454,8 +473,16 @@ file_editor.save = Zapisz
 file_editor.save_as = Zapisz jako...
 file_editor.save_warning = Zapisać zmiany przed zamknięciem ?
 file_editor.cannot_write = Nie mogę zapisać pliku.
-text_editor.cut = Wytnij
-text_editor.paste = Wklej
+text_editor.edit = $[text_viewer.edit]
+text_editor.copy = $[text_viewer.copy]
+text_editor.select_all = $[text_viewer.select_all]
+text_editor.view = $[text_viewer.view]
+text_editor.find = $[text_viewer.find]
+text_editor.find_next = $[text_viewer.find_next]
+text_editor.find_previous = $[text_viewer.find_previous]
+text_editor.spaces_to_tabs = Replace spaces with tabs
+text_editor.tabs_to_spaces = Replace tabs with spaces
+text_editor.tab_size = $[text_viewer.tab_size]
```

Comment:
do we really need lines 476-482 and 485? they seem redundant since they exist in dictionary.properties